docs/src/user-docs/quick-start/clp-text.md

```
@@ -29,6 +29,19 @@ If CLP fails to start (e.g., due to a port conflict), try adjusting the settings
 
+:::{warning}
+**Do not comment out or remove the `package` block in `etc/clp-config.yml`**, as the default storage
+engine is `clp-s`, which is optimized for JSON logs rather than unstructured text logs.
+
+To use `clp-text`, the `package` block should be configured as follows:
+
+```yaml
+package:
+  storage_engine: "clp"
+  query_engine: "clp"
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)